### ISSUE TITLE
Track railway:station and aeroway:aerodrome

### ISSUE THREAD

**Issue by dragonbane0** [OSMNames/OSMNames#169]
It might be a good idea to track the following 2 data types as well:

https://nominatim.openstreetmap.org/details.php?place_id=49784637
https://nominatim.openstreetmap.org/details.php?place_id=94628579

They could be grouped under places since it feels like these are common places people search for and expect to find. If I get the time I might try it myself since I love this project and it is of great use to my current project

**Comment by sfkeller** [comment]
Good idea. In fact, Public station names very often take over local place names.

Unfortunately, there's only minimal maintenance resources here. So we'd be happy to accept pull requests.

**Comment by philippks** [comment]
hey @dragonbane0 

Very good idea! The place to add the additional types would be [here](https://github.com/OSMNames/OSMNames/blob/master/data/import/mapping.yml). Maybe you find some time to add the proposed types and test it with a small dataset of a country?

**Comment by philippks** [comment]
hey @dragonbane0 

Very good idea! The place to add the additional types would be [here](https://github.com/OSMNames/OSMNames/blob/master/data/import/mapping.yml). Maybe you find some time to add the proposed types and test it with a small dataset of a country?

**Comment by dragonbane0** [comment]
I took the last weekend to study the code and got a pretty good grasp on it now. Currently in the process of adapting it to my needs which includes things beyond the scope of this project like post codes that have been mapped for Germany by OSM contributors and tourist attractions and hotels. It works amazingly well and search performance remains up there. When I'm done I will make a PR with just the aerodrome and station type included. Works pretty much perfectly without much adjustments (at least for Germany which I tested so far). Other types such as bus_stops are more problematic since they overlap with street names and there are many duplicates (each individual stop of course has an entry with the same name) which needs filtering. But stations and airports are unique enough that I think adding them to this project will be a definitive improvement for sure
